### ISSUE TITLE
Idlechan function

### ISSUE THREAD

**Issue by joshcoales** [joshcoales/Hallo#57]
Would be pleasant to rebuild this sometime

fnn_chat_to_s()
        'chats to _S'
        messages = ['so, how are things?','is your end of the world holding up ok?',"let's talk about deer!","boop.","beep.","I'll be honest, this place is dead.","Fuck snow!","I guess everyone's asleep.","clap... clap... _clap_","I'm watching you, robot.","What do you think of that D000242 guy?","What do you think of that spangle kid?","Butts.","I choose you!","How old are you in milli-Americas?"]

_tumbleweed_

and possibly "deer deer deer"

Basically, it should be possible to set a number of messages which can be returned
<channel>
 <enabled>False</enabled>
 <name>#ecco-the-dolphin</name>
 <idle_time>3600</idle_time>
 <message>_S: so, how are things?</message>
 <message>_S: is your end of the world holding up ok?</message>
...
</channel>

**Comment by joshcoales** [comment]
No real demand
